config/environments/development.rb

```diff
@@ -33,17 +44,23 @@
   config.eager_load = false
 
   require 'rack-webauth/test'
-  config.middleware.use(
-    Rack::Webauth::Test,
-    user: Settings.WEBAUTH.USER,
-    mail: Settings.WEBAUTH.MAIL,
-    ldapprivgroup: Settings.WEBAUTH.LDAPPRIVGROUP,
-    suaffiliation: Settings.WEBAUTH.SUAFFILIATION,
-    displayname: Settings.WEBAUTH.DISPLAYNAME,
-    ldapauthrule: Settings.WEBAUTH.LDAPAUTHRULE
-  )
+  if Settings.WEBAUTH
```

Comment:
Yes agree... we should then in argo-dev sul-configs configure `config/development.yml` to have the deployed config values. the config gem will then use that as the webauth settings for the deployed app.